addons/mail/static/src/models/thread/thread.js

```diff
@@ -1608,14 +1694,31 @@ function factory(dependencies) {
             compute: '_computeLastCurrentPartnerMessageSeenByEveryone',
             dependencies: [
                 'partnerSeenInfos',
-                'orderedMessages',
+                'orderedNonTransientMessages',
```

Comment:
alphabetical order (btw wasn't respected before diff, but not a reason not to improve)